### DISCUSSION TITLE
display substream in web-gui main page?

### DISCUSSION THREAD

**Discussion by Cold-Lemonade** [blakeblackshear/frigate#7528] [Ask A Question] [answered]
If all detect streams are already being decoded by Frigate for detection, then how much additional computing power would be required to display these live detect streams in the Frigate main gui page? Is there a limitation on preventing this functionality? Could it be added as an option that could be chosen in the Frigate config file?

This feature would be really nice to have and something I miss about Nx Witness. I understand that the birdseye page is meant to show the live view of any camera on which motion is detected using the stream designated for detect. But it isn't the same as showing all the live streams whether or not motion has been detected.

**Comment by NickM-27** [comment, marked as answer]
https://github.com/blakeblackshear/frigate/pull/7071

**Reply by Cold-Lemonade** [reply]
Exciting! Thank you.
